varats/varats/data/reports/blame_report.py

```diff
@@ -516,6 +516,62 @@ def generate_degree_tuples(
     return list(degree_dict.items())
 
 
+def generate_lib_dependent_degrees(
+    report: tp.Union[BlameReport, BlameReportDiff]
+) -> tp.Dict[str, tp.Dict[str, tp.List[tp.Tuple[int, int]]]]:
+    """
+    Args:
+        report: The blame report
+
+    Returns:
+        Map of tuples (degree, amount) categorised by their corresponding
+        library name to their corresponding base library name.
+    """
+    categorised_degree_dict: tp.Dict[str, tp.Dict[str, tp.Dict[int, int]]] = {}
+
+    for func_entry in report.function_entries:
+        for interaction in func_entry.interactions:
+            base_repo_name = interaction.base_commit.repository_name
+            tmp_degree_of_libs: tp.Dict[str, int] = {}
+
+            if not categorised_degree_dict:
+                categorised_degree_dict[base_repo_name] = {}
+
+            for inter_hash in interaction.interacting_commits:
+                inter_hash_repo_name = inter_hash.repository_name
+
+                if inter_hash_repo_name \
+                        not in categorised_degree_dict[base_repo_name]:
```

Comment:
```suggestion
                if (inter_hash_repo_name 
                        not in categorised_degree_dict[base_repo_name]):
```
With `()` you get sometimes better line breaks for long conditions.

PS: run format after applying suggestion